clients/src/main/java/org/apache/kafka/common/config/AbstractConfig.java

```diff
@@ -226,6 +227,10 @@ public Password getPassword(String key) {
         return keys;
     }
 
+    public boolean originalsContainsKey(String configKey) {
```

Comment:
AbstractConfig is sadly a public class. Can we do the PR without this?

Reply:
Oops! OK, reverted.